.github/workflows/npm-types.yml

```diff
@@ -22,13 +22,15 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v3
-      - uses: bazelbuild/setup-bazelisk@v2
```

Comment:
where does bazel come from  now?

Reply:
The default runner images include bazelisk, I mentioned it in the commit description

Reply:
oh, can we add this to a PR description? I assume that commit descriptions will be lost after squash.

Reply:
Done!